docs/troubleshooting/README.md

```diff
@@ -1 +1,114 @@
-TBA.
\ No newline at end of file
+# Troubleshooting guide
+
+## Overview:
+
+This TSG is meant to help you troubleshoot issues with the Fleet APIs.
+
+## Cluster Resource Placement:
+
+Internal Objects to keep in mind when troubleshooting CRP related errors on the hub cluster:
+ - **ClusterResourceSnapshot**
+ - **ClusterSchedulingPolicySnapshot**
+ - **ClusterResourceBinding**
+ - **Work** 
+
+please read the API reference for more details about ech object https://github.com/Azure/fleet/blob/main/docs/api-references.md
+
+### How can I debug when my CRP status is ClusterResourcePlacementScheduled condition status is set to "False"?
+
+We need to take a look at the **ClusterSchedulingPolicySnapshot** status to figure out why the scheduler could not schedule the resource for the placement policy specified.
+
+### How to find the latest ClusterSchedulingSnapshot resource?
+
+We need to have ClusterResourcePlacement's name **{CRPName}**, replace **{CRPName}** in the command below,
+
+```
+$ kubectl get clusterschedulingpolicysnapshot -l kubernetes-fleet.io/is-latest-snapshot=true,kubernetes-fleet.io/parent-CRP={CRPName}
+```
+
+### How can I debug when my CRP status is ClusterResourcePlacementSynchronized condition status is set to "False"?
```

Comment:
could you please list when the sync will be set as "false" first?

 providing the troubleshooting steps is helping them to validate the case and find the action item.

Here, you only provide the validation part, but not the action item.